### ISSUE TITLE
x !in Iterable gets negated

### ISSUE THREAD

**Issue by Sebmaster** [decaffeinate/decaffeinate#1242]
Decaffeinate is producing the wrong JavaScript based on my CoffeeScript input:

```coffee
if 1 !in [1]
    console.log('woops')
```

https://decaffeinate-project.org/repl/#?evaluate=true&stage=full&code=if%201%20!in%20%5B1%5D%0A%20%20%20%20console.log('woops')

I get this output:

```js
if ([1].includes(1)) {
    console.log('woops');
}
```

Here's what I expect it to be instead:

```js
const x = 1;

if (![1].includes(x)) {
    // do something
    console.log;
}
```

**Comment by eventualbuddha** [comment]
Interesting. Looks like `not in` works but `!in` doesn’t. The `isNot` property of the `InOp` is `false` when it should be `true`. Should be an easy fix.

**Comment by Sebmaster** [comment]
Ah yes, there's a similar crashing problem with `!instanceof` as well.

**Comment by eventualbuddha** [comment]
Fixed in v4.0.1.
